modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -10704,6 +10704,7 @@ no-outstanding-invites=No outstanding invites.
 no-page-template-could-be-imported=No page template could be imported.
 no-pages=No Pages
 no-pages-were-found-that-matched-the-keywords-x=No pages were found that matched the keywords: {0}.
+no-parameter-were-found=No parameter were found.
```

Comment:
@danvb112 shouldn't it be "No parameters were found"? If we are referring to just one parameter, then it should be "No parameter was found".